org-formation/organization.yaml

```diff
@@ -444,60 +444,3 @@ Organization:
                 'aws:RequestedRegion':
                   - us-east-1
                   - us-east-2
```

Comment:
Do you mean to include `us-east-2`? I see that its in the sid name on L436 too

Reply:
actually those were read by ofn and included in this file.  I'm going to move all of those SCPs into the `010-scps` folder like i did with RestrictUnusedRegionsSCP.  will do it in follow on PR.